apps/web/src/components/Menu/UserMenu/WalletInfo.tsx

```diff
@@ -31,7 +31,8 @@ import { useDomainNameForAddress } from 'hooks/useDomain'
 import { useState } from 'react'
 import { isMobile } from 'react-device-detect'
 import { getBlockExploreLink, getBlockExploreName } from 'utils'
-import { Address, useBalance } from 'wagmi'
+import { Address } from 'viem'
```

Comment:
May import type only here
```suggestion
import type { Address } from 'viem'
```